### DISCUSSION TITLE
DietPi-Software | Add shell selection with zsh and fish

### DISCUSSION THREAD

**Discussion by MichaIng** [MichaIng/DietPi#6118] [Software Request] [answered]
Quite a significant number if Linux users use/like zsh, quite some fish. So related install options and a shell selection in `dietpi-software` would be nice, which installs the selected shell, adds [an adapted DietPi login script](https://github.com/MichaIng/DietPi/pull/5211) and makes it the default shell for `root` and `dietpi` (as long as it exists) users.

**Comment by therealresonix** [comment]
@MichaIng Would be nice to have an update on this; Would be cool if you could add it to one of the next releases.

**Comment by oxoocoffee** [comment]
Any updates on this request?

**Comment by ndrew222** [comment, marked as answer]
I've drafted up a pull request #7452 
Solution has been inspired by #5211
